includes/blocks/class-kadence-blocks-row-layout-block.php

```diff
@@ -215,15 +215,15 @@ public function get_template_columns( $css, $columns, $layout, $inner_selector,
 	/**
 	 * Builds CSS for block.
 	 *
-	 * @param array $attributes the blocks attributes.
+	 * @param array  $attributes the blocks attributes.
 	 * @param string $css the css class for blocks.
 	 * @param string $unique_id the blocks attr ID.
 	 */
 	public function build_css( $attributes, $css, $unique_id ) {
 		$css->set_style_id( 'kb-' . $this->block_name . $unique_id );
 		if ( ! empty( $attributes['kbVersion'] ) && $attributes['kbVersion'] > 1 ) {
 			$base_selector = '.kb-row-layout-id' . $unique_id;
-			$inner_selector = '.kb-row-layout-id' . $unique_id . ' > .kt-row-column-wrap';
+			$inner_selector = '.kb-row-layout-id' . $unique_id . ' > .kt-row-column-wrap.kt-row-column-wrap'; // Doubling .kt-row-column-wrap here to increase specificty to avoid conflict with alignment.
```

Comment:
Can you explain how to recreate the issue, I want to look for a different way to fix the problem, this will effect all styles and I would like to avoid that

Reply:
ya this one felt sketchy to me too..

you set a custom left or right padding on a row layout, then set the alignment to full width. On the fronted the custom padding won't apply.

Reply:
fixed this the other way by lowering the specificity of the other selector on alignment